docs/generate-time-series/cli.rst

```diff
@@ -32,96 +32,81 @@ Crash is available as `Pip`_ package. `Install`_ it like this:
 We designed the commands in this tutorial to be run directly from the `command
 line`_ so that you can experiment with them as you see fit.
 
+You should also ensure you have the command line tools `curl`_ and `jq`_ installed.
+
 .. NOTE::
 
     This tutorial should work in most POSIX-compatible environments (e.g.,
     Linux, macOS, and Windows Cygwin). Please `let us know`_ if you run into
     issues.
 
 
-Process ``uptime`` to get your system load
-==========================================
-
-The `uptime`_ command is a convenient way to get your current system load.
-
-The exact output varies a little from system to system, but the ``uptime``
-command will typically display something like this:
+Using ``curl`` to get the current position of the ISS
+=====================================================
 
-.. code-block:: console
-
-    $ uptime
-    16:58  up 3 days,  5:45, 4 users, load averages: 3.15 4.51 5.06
+You can get telemetry data from `Open Notify`_, a third-party service that
+provides a simple API to consume data from NASA (specifically, the current
+location of the International Space Station). The endpoint for this data is
```

Comment:
Imho "these data", data is plural. Or "this set of data", etc.

Reply:
disagree. data is a mass noun! :)